packages/next/build/webpack-config.ts

```diff
@@ -753,6 +753,31 @@ export default async function getBaseWebpackConfig(
     )
   }
 
+  const getPackagePath = (name: string, relativeToPath: string) => {
+    const packageJsonPath = require.resolve(`${name}/package.json`, {
+      paths: [relativeToPath],
+    })
+    // Include a trailing slash so that a `.startsWith(packagePath)` check avoids false positives
+    // when one package name starts with the full name of a different package.
+    // For example:
+    //   "node_modules/react-slider".startsWith("node_modules/react")  // true
+    //   "node_modules/react-slider".startsWith("node_modules/react/") // false
+    return path.join(packageJsonPath, '../')
+  }
+
+  // Packages which will be split into the 'framework' chunk.
+  // Only top-level packages are included, e.g. nested copies like
+  // 'node_modules/react-scroll-parallax/node_modules/prop-types' are not included.
+  const topLevelFrameworkPaths = [
+    getPackagePath('react', dir),
+    getPackagePath('react-dom', dir),
```

Comment:
prop-types seem to be missing here?

It was part of the original regexp

Reply:
Also `react` and `react-dom` depend on `object-assign` so that might also need to be added here, even if it wasn't part of the regexp before.

Reply:
Added

Reply:
For what it's worth, this error is raised when `getPackagePath('prop-types', dir)` is added to the array:

```
```